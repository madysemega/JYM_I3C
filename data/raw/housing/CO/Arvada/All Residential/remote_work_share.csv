predictions_remote_work_share
7.184384393692016
3.5129144155979155
6.0147651767730705
3.1937815940380094
2.470793881416321
5.616189277172088
4.554770174026489
3.8681198477745053
5.744464967250823
5.950551283359527
2.4158572924137114
5.825477986335754
4.958895614147186
1.964733857512474
5.044219501018524
3.309856297969818
3.024707942008972
4.407198503017425
3.528848341703415
2.017078232169151
3.1861693906784057
4.304021165370941
4.953925395011901
4.921657392978668
5.406132519245148
4.17581077337265
3.8678064966201777
