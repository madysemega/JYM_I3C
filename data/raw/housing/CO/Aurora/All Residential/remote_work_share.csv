predictions_remote_work_share
10.170499155521393
7.564339802265168
6.970026079416275
7.655484107732773
6.128996983766556
6.829985467195511
8.23335622906685
6.77725292801857
7.677154794931412
7.632457295656204
8.253933069705964
10.415080044269562
9.469813916683197
7.196434894800186
7.928133014440537
7.051259843111039
6.213228348493576
9.615837488174439
8.501946178674698
6.558007088899613
9.511488387584686
6.727721879482269
8.283972177505493
8.63018567919731
9.371561727523805
7.102332541942596
8.275055185556411
