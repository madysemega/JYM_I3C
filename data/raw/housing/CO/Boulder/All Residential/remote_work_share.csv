predictions_remote_work_share
15.282467151880265
16.806278967857363
18.536740393638613
16.722232080698014
17.5005773639679
16.648074642419818
14.550851425528528
16.398275675177576
15.782915202975275
14.432562295794488
12.835232436656954
15.683877134323122
15.1089493650198
16.173369733691217
15.541662851572038
17.138136396408083
17.74841889381409
16.989088126420977
15.918898740410807
15.24235879480839
16.63502023935318
14.365976369976998
14.067050372362138
14.726278101205828
15.370286366939547
16.81817233681679
16.6213584625721
