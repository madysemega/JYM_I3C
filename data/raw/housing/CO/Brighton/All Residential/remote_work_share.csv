predictions_remote_work_share
11.318110431432723
6.991420859098434
7.972923148870468
8.141428906917572
7.3136460310220714
7.532991145849228
10.591648753881454
10.31832346200943
6.918297157287597
9.967572734355926
4.001350725889206
10.700558631420135
11.999599120616912
4.917577899098396
11.185303058624267
11.53030928492546
10.938939843177794
11.116622017621994
6.249251037240028
7.964602112174034
11.303062448501587
6.316493353843689
11.751083883047103
11.962925798892973
9.915832947492598
12.023291001319885
6.925492511987685
