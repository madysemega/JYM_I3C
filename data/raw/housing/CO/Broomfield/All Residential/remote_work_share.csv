predictions_remote_work_share
18.526909288167957
11.501842688918115
18.189961155653005
18.337038120031362
8.167797630429268
13.961441185474397
19.51990573048592
12.103678601384164
14.594321908950807
10.539818480014802
10.75350327730179
14.724562915563585
12.060357954502107
11.103372266292574
19.66589300870896
18.244604929685597
9.523046230673792
16.846811547279362
15.665383222103122
9.945815836787226
19.853971203565603
9.346250274181367
16.844538130760196
15.980506964921954
15.506587531566622
16.92754236698151
13.974163731336596
