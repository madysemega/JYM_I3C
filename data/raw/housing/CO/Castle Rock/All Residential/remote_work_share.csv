predictions_remote_work_share
4.492505269050597
4.925618228912353
5.3774407196044915
3.9824710607528684
4.276262798309326
3.2024041843414306
4.189202184677123
3.535428280830383
4.13314299583435
3.7900137281417843
3.7133582448959346
4.404947657585144
4.737879667282104
4.481721634864806
5.005413126945495
4.336317944526672
4.122345142364502
4.096107177734375
3.75360152721405
3.369571504592895
4.730529313087463
4.740287466049194
4.485623812675476
4.631966314315796
4.539803686141967
5.108019819259643
4.39190721988678
