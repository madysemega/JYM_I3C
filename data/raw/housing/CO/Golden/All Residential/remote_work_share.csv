predictions_remote_work_share
9.25610304892063
10.160129754543304
11.378019005060194
10.535311336517333
9.614878686666488
11.28557617664337
8.805868435502052
9.730546216368674
11.028771560192107
10.058776329755784
8.722382907271385
10.584283974170683
11.080578382015227
10.601321034431455
10.811338260173796
10.564013289213179
11.555290710926055
10.979243392944335
9.613335002660751
8.77040364742279
10.313802514076231
11.324072411060332
10.615169944763181
9.711436367034912
9.978146494030952
12.408414559364317
11.468856697082519
